XYZDrivers/src/main/java/com/xyzdrivers/models/InsertClaim.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file InsertClaim.java
+ * @author Nathan
+ * @created 02/11/17
+ * @modified 02/11/17
+ * @notes -
+ */
+
+package com.xyzdrivers.models;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class InsertClaim {
+
+    Connection con;
+
+    public void InsertClaim(Claim c) {
+
+        try {
+
+            Class.forName("org.apache.derby.jdbc.ClientDriver");
+            con = DriverManager.getConnection("jdbc:derby://localhost:1527/xyzdrivers");

```

Comment:
Move the code to create the connection outside of the class. Pass the connection in through the constructor. The service should only care that it has a connection, not how it's created.